Source/SG/SG.cs

```diff
@@ -172,10 +174,31 @@ public static void DownloadWaveform(NIRfsg rfsgHandle, ref Waveform waveform)
             NIRfsgPlayback.StoreWaveformRuntimeScaling(rfsgPtr, waveform.WaveformName, -1.5);
```

Comment:
should this be moved to a waveform property but defaulted to -1.5 in the readfromfile? what is the file has a different runtime scaling?